shap/benchmark/plotting.py

```diff
@@ -0,0 +1,57 @@
+import matplotlib.pyplot as plt 
+import pandas as pd 
+import numpy as np 
+
+root = 'C:/Users/maggiewu/Desktop/'
```

Comment:
hmm seeing these kind of constants is concerning, any way to use relative path or make the code more generic?

Reply:
File will be removed. Meant to only be a temporary plotting file. Thank you for catching.